### ISSUE TITLE
Can't mount block volume in /

### ISSUE THREAD

**Issue by gbouras** [oracle/learning-library#356]
/*
 *Following directions in
 *https://github.com/oracle/learning-library/blob/master/oci-library/L100-   
 *LAB/Block_Volume/Block_Volume_HOL.md
 *with the exception of initially trying to mount the device to /u01 in the root
 *level directory which doesn't work. However mounting to /mnt/disk1 does.
 *Is this expected behavior?
 */
[root@cloud1 /]# mount /dev/oracleoci/oraclevdb /u01

[root@cloud1 /]# df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        7.2G     0  7.2G   0% /dev
tmpfs           7.3G     0  7.3G   0% /dev/shm
tmpfs           7.3G  8.6M  7.3G   1% /run
tmpfs           7.3G     0  7.3G   0% /sys/fs/cgroup
/dev/sda3        39G  2.1G   37G   6% /
/dev/sda1       200M  9.6M  191M   5% /boot/efi
tmpfs           1.5G     0  1.5G   0% /run/user/1000
//Operation as you can see fails

[root@cloud1 /]# mount /dev/oracleoci/oraclevdb /mnt/disk1
[root@cloud1 /]# df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        7.2G     0  7.2G   0% /dev
tmpfs           7.3G     0  7.3G   0% /dev/shm
tmpfs           7.3G  8.6M  7.3G   1% /run
tmpfs           7.3G     0  7.3G   0% /sys/fs/cgroup
/dev/sda3        39G  2.1G   37G   6% /
/dev/sda1       200M  9.6M  191M   5% /boot/efi
tmpfs           1.5G     0  1.5G   0% /run/user/1000
/dev/sdb         50G   53M   47G   1% /mnt/disk1
//But using a sub directory operation succeeds

//Both directories exist:
[root@cloud1 /]# ls -d /u01
/u01
[root@cloud1 /]# ls -d /mnt/disk1
/mnt/disk1

**Comment by flavio-santino** [comment]
Hi Greg, this is not the expected behavior. What is the OS version that are using?

**Comment by gbouras** [comment]
> Hi Greg, this is not the expected behavior. What is the OS version that are using?
Hello Flavio,
This was on Oracle Linux version 7.6 OS Build# 2019.01.17.-0 
I destroyed that instance and re-created two more since this post, with 4 additional block volumes and then mounted two on each instance. I did not encountered this issue again. So I don't really have an explanation. Maybe fussing around too much with gdisk/fdisk, I have no idea. But the output above was sequential and nothing was cut out. I replaced the hostname to shorten it but that's all.
